spec/serializers/api/v3/statement_serializer_spec.rb

```diff
@@ -0,0 +1,82 @@
+# spec/serializers/api/v1/statement_serializer_spec.rb
+
+require "rails_helper"
+
+RSpec.describe API::V3::StatementSerializer, type: :serializer do
+  let(:subject) { described_class }
+  let(:cohort) { create(:cohort) }
+  let(:statement) { create(:statement, cohort:) }
+
+  it "serializes the `id`" do
+    response = subject.render_as_hash(statement)
```

Comment:
You can make this into a `let`, it'll dry up your code:
```ruby
let(:response) { subject.render_as_hash(statement) }
```

Reply:
interesting thoughts here, I know in ECF we've approached this using the DRY'ed up version, and I've definitely found it hard to maintain and follow. Personally, Im happy with having tests that are self contained and can be easily understood or maintained at the price of duplication. 

The problem with ECF and why we couldn't do that is the large setup required, maybe here we won't be hitting the same issues. The only extra suggestion from me is the risk of not persisting values on objects for test speed which sometimes has unexpected behaviour so something to watch out for. Happy to hear further thoughts as this is a personal preference ofcourse

Reply:
@pmanrubia 
It'll just reduce the noise, the `response` line is not important for the test, its exactly the same for every test. As long as we keep the DRY within the same file I don't see it being an issue.